src/pages/LogIn/Login.js

```diff
@@ -2,8 +2,100 @@ import React, { Component } from 'react';
 import './Login.scss';
 
 class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      pw: '',
+    };
+  }
+
+  handleInput = e => {
+    const { value, name } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  postLogin = () => {
+    const { id, pw } = this.state;
+    if (
+      this.state.id.includes('@') &&
+      this.state.id.includes('.com') &&
+      this.state.pw.length >= 8
+    ) {
+      fetch('http://172.30.1.25:8000/users/signin', {
+        method: 'POST',
+        body: JSON.stringify({
+          email: id,
+          password: pw,
+        }),
+      })
+        .then(response => response.json())
+        .then(response => {
+          if (response.MESSAGE === 'SUCCESS') {
+            this.props.history.push('/mainPage');
+          } else if (response.MESSAGE === 'INVALID_USER') {
+            alert('이메일과 비밀번호를 다시 확인해주세요');
+            this.setState({ id: '' });
+            this.setState({ pw: '' });
+          }
+        });
+    } else {
+      alert('이메일 및 비밀번호를 다시 입력해주세요');
+      this.setState({ id: '' });
+      this.setState({ pw: '' });
```

Comment:
```suggestion
      this.setState({ id: '', pw: '' });
```
객체 형태이므로 한번에 모아서 처리할 수 있습니다

Reply:
넵 !